src/cat-data-service/src/state/jorm_mock.rs

```diff
@@ -1,4 +1,4 @@
-use crate::types::jorm_mock::{
+use crate::legacy_service::types::jorm_mock::{
```

Comment:
//! Mocking the Jormungandr Node for the Webbased Voting App Testing.
//!
```suggestion
use crate::legacy_service::types::jorm_mock::{
```